cmd/snap-update-ns/change.go

```diff
@@ -667,23 +694,27 @@ func neededChanges(currentProfile, desiredProfile *osutil.MountProfile) []*Chang
 
 		sort.Strings(allMimicCreationDirs)
 
+		logger.Debugf("adding mimics and their entries")
+
 		for _, mimicDir := range allMimicCreationDirs {
 			// make sure to sort the entries for each mimic dir in a consistent
 			// order
 			entries := entriesForMimicDir[mimicDir]
 			sort.Sort(byOriginAndMagicDir(entries))
+			logger.Debugf("mimic creation dirs")
+			for _, en := range allMimicCreationDirs {
```

Comment:
this looks wrong here, was it meant to be before the for loop? or to be entries?